components/widgets/datetime/multiple-timezones.js

```diff
@@ -0,0 +1,84 @@
+import { Component } from 'react'
+import styled from 'styled-components'
+import Widget from '../../widget'
+import moment from 'moment-timezone'
+import Table from '../../table'
+import { object, string, array, number, boolean } from 'yup'
+
+const TimeItem = styled.div`
+  font-size: 3em;
+  text-align: center;
+  text-align: right;
+`
+
+const schema = object().shape({
+  format24: boolean(),
+  timeZones: array(object({
+    name: string().required(),
+    flag: string().required()
+  })).required().max(5),
+  interval: number()
+})
+
+export default class MultipleTimezones extends Component {
+  static defaultProps = {
+    interval: 1000 * 10,
+    timeZones: [{ name: 'Asia/Colombo', flag: 'India.png' }, { name: 'Europe/Copenhagen', flag: 'Denmark.png' }],
+    format24: false
+  }
+
+  state = {
+    date: new Date(),
+    error: false,
+    loading: true
+  }
+
+  componentDidMount () {
+    const { interval } = this.props
+
+    schema.validate(this.props).then(() =>
+      this.setState({ loading: false }),
+    this.timeout = setTimeout(() => this.setState({ date: new Date() }), interval))
+      .catch((err) => {
+        console.error(`${err.name} @ ${this.constructor.name}`, err.errors)
+        this.setState({ error: true, loading: false })
+      })
+  }
+
+  componentWillUnmount () {
+    clearTimeout(this.timeout)
+  }
+
+  render () {
+    const { date, error, loading } = this.state
+    const { timeZones, format24 } = this.props
+
+    function Flag (props) {
+      let image = '/static/flags/' + props.flag
```

Comment:
* I'd like to avoid the static images. I think https://github.com/wiredmax/react-flags or https://github.com/smucode/react-world-flags would be a better choice.
* `const` instead of `let`
* Template literals vs. concatenated strings -> We should use template literals.

Reply:
https://github.com/wiredmax/react-flags seems like a good way to accomplish this, but you would still have to manually copy the flags from node_modules/react-flags/vendor/flags to static/flags. 

Are there anyway to allow the flags to be served as static content directly from the node_modules path? 

Then this module can reach the flag, without asking the user to copy the flags manually

                  <Flag
                    name={zone.flag}
                    format="png"
                    pngSize={64}
                    shiny={true}
                    basePath='node_modules/react-flags/vendor/flags'
                  />

Reply:
Fixed for now, but it requires the user to manually copy the files